src/test/java/org/lookout_studios/meals_management_system/meals_management_system/MealsManagementSystemApplicationTests.java

```diff
@@ -8,6 +8,12 @@ class MealsManagementSystemApplicationTests {
 
 	@Test
 	void contextLoads() {
+		EstablishConnectionWithDatabaseTest 
+		connectionWithDatabaseFileReferenceSetUp = new EstablishConnectionWithDatabaseTest () ;
+	connectionWithDatabaseFileReferenceSetUp.setUp ();
+		EstablishConnectionWithDatabaseTest
+		connectionWithDatabaseFileReference = new EstablishConnectionWithDatabaseTest () ;
+	connectionWithDatabaseFileReference.testMockDBConnection ();
```

Comment:
You don't need to invoke the tests in a separate class. With these lines you test the same code twice: once directly in `EstablishConnectionWithDatabaseTest` and here again, invoking the same test methods. You can delete these lines.